activiti-api/activiti-api-task-runtime/src/main/java/org/activiti/api/task/runtime/TaskRuntime.java

```diff
@@ -133,4 +134,12 @@
     
     List<String> userCandidates(String taskId);
     List<String> groupCandidates(String taskId);
+
+    /**
+     * Assign a task with a new user
+     *  - If there is a user assigned, reassign it to the new user
+     *  - user should be a candidate-user for the task
+     *  - after the assign the task should be in assigned status
+     */
```

Comment:
I'd rephrase this a little bit to make it clear that the task should be claimed before and only the assignee can do it. Also, I don't think it's necessary to mention that mention that the task will be in the assigned status:
```
Assign a task that has been claimed before to a different user. Only the current assignee can perform the action and the new assignee should be part of the candidate users for this task.
```
What do you think?

Reply:
Yes, I agree with you. Thanks!